src/main/java/fr/spoonlabs/flacoco/FlacocoMain.java

```diff
@@ -1,20 +1,23 @@
 package fr.spoonlabs.flacoco;
 
-import java.io.File;
-import java.util.Map;
-import java.util.concurrent.Callable;
-
 import fr.spoonlabs.flacoco.core.config.FlacocoConfig;
 import fr.spoonlabs.flacoco.localization.spectrum.SpectrumFormula;
 import fr.spoonlabs.flacoco.localization.spectrum.SpectrumRunner;
 import picocli.CommandLine;
 import picocli.CommandLine.Command;
 import picocli.CommandLine.Option;
 
+import java.io.File;
+import java.util.Map;
+import java.util.concurrent.Callable;
+
 @Command(name = "FlacocoMain", mixinStandardHelpOptions = true, version = "0.0.1", description = "Flacoco: fault localization")
 public class FlacocoMain implements Callable<Integer> {
 	static boolean log = true;
 
+	@Option(names = { "-w", "--workspace"}, description = "Path to the workspace directory of flacoco.", defaultValue = "./")
```

Comment:
I wonder who it would work when we use Flacoco with a Jar: it will not be a workspace

Reply:
I'm not sure I understood. I just ran the jar with that option and it works fine.

That option, for now, is only passed to `test-runner`, which will then save and load serialized files inside of it.

How could running it through the packaged jar result in different behavior?

Reply:
Ok, perfect. I misunderstood. Sorry :)